bundle/manifests/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -280,6 +275,7 @@ spec:
                       set up for the plugin Deployment.
                     properties:
                       maxReplicas:
+                        default: 3
```

Comment:
:thinking: are these changes related to your environment ?

I see FlowCollectorHPA.MaxReplicas defined using:
```go
MaxReplicas int32 `json:"maxReplicas" protobuf:"varint,3,opt,name=maxReplicas"`
```
instead of using annotations like:
```go
//+kubebuilder:validation:Minimum=0
//+kubebuilder:default:=3
```

We should create a followup to harmonize these if it's the case

Reply:
No, they aren't. But the generated flowcollectors manifest was outdated so when I regenerated mine, the old changes took effect.

Maybe we should add a test target that verifies that the manifests are always up to date.